Appserver/Controllers/TimesheetController.cs

```diff
@@ -121,22 +121,53 @@ public IActionResult Submit(int id)
                 childform.FromJson(a);
                 textractform.AddPages(childform);
             }
-            var tsf = (TimesheetForm)AbstractFormObject.FromTextract(textractform);
-
-            tsf.id = id;
+            var ts = (TimesheetForm)AbstractFormObject.FromTextract(textractform);
 
-            var dbutil = new FormToDbUtil(_subcontext, _context);
-            Timesheet ts = dbutil.PopulateTimesheet(tsf);
-            dbutil.PopulateTimesheetEntries(tsf, ts);

```

Comment:
Should those db utilities be removed [from the IDD namespace] if they aren't being used any longer?